src/detect-replace.c

```diff
@@ -69,6 +69,8 @@ static int DetectReplacePostMatch(ThreadVars *tv,
 void DetectReplaceRegister (void)
 {
     sigmatch_table[DETECT_REPLACE].name = "replace";
+    sigmatch_table[DETECT_REPLACE].desc = "only be used in ips. changes the content it follows into another";
```

Comment:
s/in ips/in IPS-mode/g.
Replaces packet content with the value in this keyword. Or something.